constants/interface-routes/elevate-project/configs.json

```diff
@@ -168,6 +168,62 @@
 			],
 			"service": "project"
 		},
+		{
+			"sourceRoute": "/project/v1/userProjects/pushSubmissionToTask/:id",
+			"type": "POST",
+			"priority": "MUST_HAVE",
+			"inSequence": false,
+			"orchestrated": false,
+			"targetPackages": [
+				{
+					"basePackageName": "project",
+					"packageName": "elevate-project"
+				}
+			],
+			"service": "project"
+		},
+		{
+			"sourceRoute": "/project/v1/userProjects/pushSubmissionToTask",
```

Comment:
@praveenKDass why do we need this route? Isn't it always going to be with id? Same for below?